README.md

```diff
@@ -33,9 +33,9 @@ The best approach to install `docker-compose` is to follow the offical guide [he
 ## Usage
 
 ### Development Environment
-To start developing, you must create a file `.env` with environment variables, which are explained in more detail [below](#env-file-specification).
+To start developing, copy `.env.example`, creating the local config file (this can be achieved by running `cp .env.example .env.local`). Then, fill in the variable values, as explained in `.env.example`file.
```

Comment:
Same suggestion listed above, using `.env` or even `.env.test` for documentation and default values might be better than having a dedicated file for that which might become outdated.